samples/02_power_users_developers/openstreetmap_exploration.ipynb

```diff
@@ -0,0 +1,1338 @@
+{
```

Comment:
<ul><li>in the environment without <em>ArcPy, </em>when I first ran this cell I receive a value of -685 Square Kilometers</li><li>I installed the <em>shapely </em>package and that corrected it initially, but now currently I receive negative values for the area queries above and the population density calculations below.</li><li>from research I've done this is usually because of ring ordering in the polygons, but I'm not sure what to add  to my environment or how to correct it</li></ul><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/551/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>